times_reader/transforms.py

```diff
@@ -1908,6 +2147,7 @@ def timeslices_table(
     return result
 
 
+# TODO: looks like there is a similar function in main. Should we delete one?
```

Comment:
Yes, let's delete the one in main. Does anyone remember why we're converting everything to string before we dump tables? Feels like a hack to patch over some other issue (and is probably related to the trouble we're having dropping NA values when columns aren't typed properly).